build.gradle

```diff
@@ -0,0 +1,102 @@
+import net.fabricmc.loom.task.RunGameTask
+
+plugins {
+  id 'java'
+  alias(libs.plugins.eclipseApt)
+  alias(libs.plugins.indra.sonatype)
+  alias(libs.plugins.nexusPublish)
+  alias(libs.plugins.ideaExt)
+}
+
+indraSonatype {
+  useAlternateSonatypeOSSHost("s01")
+}
+
+allprojects {
+  // subprojects would not configure second layer subprojects
+  if (project == rootProject) {
+    return
+  }
+
+  apply plugin: "net.kyori.indra"
+  apply plugin: "net.kyori.indra.checkstyle"
+  apply plugin: "net.kyori.indra.licenser.spotless"
+  apply plugin: "com.diffplug.eclipse.apt"
+  apply plugin: "org.jetbrains.gradle.plugin.idea-ext"
+
+  indraSpotlessLicenser {
+    licenseHeaderFile(rootProject.file("LICENSE_HEADER"))
+  }
+
+  spotless {
+    //ratchetFrom("origin/mc/${libs.versions.minecraft.get().splitToSequence('.').take(2).joinToString(".")}")
+    ratchetFrom("origin/mc/1.20")
+
+    java {
+      importOrderFile(rootProject.file(".spotless/kyori.importorder"))
+      trimTrailingWhitespace()
+      endWithNewline()
+      indentWithSpaces(2)
+      removeUnusedImports()
+    }
+  }
+
+  indra {
+    github("KyoriPowered", "adventure-platform-fabric") {
```

Comment:
heads up for us all, once this PR is merged we should remember to rename the repo and its references to adventure-platform-mod.